### PR TITLE
Add NHS ID consent questionnaire

### PR DESCRIPTION
**Pull request by mpgxvii**
- Add NHS ID consent questionnaire

### CODE REVIEW

**Review by yatharthranjan — COMMENTED** [review]
Please find some comments.

overall it is still not clear to me -

1. which study will this questionnaire be used in?
2. It mentions collection of activity and GPS data. - which, unless coming from healthkit, cannot be obtained from aRMT app, so which app will be used ?

**Comment by afolarin** [comment]
Obviously, we can't have a generic consent form if we need to mention the specifics of the study. I think for the BHF they are doing all this in one content form i.e. Study Consent + NHS-ID collection consent. It may make sense to split this into two. So we can have a generic NHS consent, which would always be paired with consent for the study (also in this way, you can consent for the study but not share your NHS-ID more easily.

File: questionnaires/nhs_id_consent/nhs_id_consent_armt.json [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+        "field_type": "info",
+        "field_label": "Consent",
+        "select_choices_or_calculations": [
+            {
+                "code": "Part1",
+                "label": "We invite you to take part in this research study. This Participant Information will explain the study to help you decide whether you want to take part. It is entirely up to you whether or not to join the study."
+            },
+            {
+                "code": "Part2",
+                "label": "The first section tells you the purpose of the study and what it involves. The second section gives more detailed information. After this we will ask you some questions to check you understand the information and are able to take part. If you decide to take part we will ask you to give your consent (permission)."
```
[the remark this answers is not in the document]
Comment (afolarin) [reply questionnaires/nhs_id_consent/nhs_id_consent_armt.json:15]: I think "able" is ok, keep the language simple.

**Comment by afolarin** [comment]
Suggest we (for demonstration purposes) have a second consent form for the Study where we can use the same fields as the BHF study. But for the NHSID stuff, we can keep that separate and modular so that it can be appended to future studies consent that are study specific?
[Study Specific Consent e.g. BHF] + [NHSID Consent]

**Review by afolarin — APPROVED** [review]

**Comment by afolarin** [comment]
What's the status here? We can discuss next wk

**Comment by mpgxvii** [comment]
> What's the status here? We can discuss next wk

@afolarin I think I made most of the changes needed from the comments above. Is this ok to merge? We can make further changes in the future if needed.

**Comment by afolarin** [comment]
Let's do this -- this is an example we can tweak with future studies, which will all doubtless have custom requirements etc.

**Pull request merged**